route53_rsaa_app.tf

```diff
@@ -0,0 +1,28 @@
+# ------------------------------------------------------------------------------
+# Resource records that support RSAA (Repository for Software Attestations and
+# Artifacts)
+
+# ------------------------------------------------------------------------------
+# Development entries
+# ------------------------------------------------------------------------------
+
+resource "aws_route53_record" "rsaa_dev_CNAME" {
+  provider = aws.route53resourcechange
+  name     = "dev.rsaa.${aws_route53_zone.cyber_dhs_gov.name}"
+  records  = ["rsaa-dev-674180318.us-gov-west-1.elb.amazonaws.com"]
+  ttl      = 300
+  type     = "CNAME"
+  zone_id  = aws_route53_zone.cyber_dhs_gov.zone_id
+}
+# ------------------------------------------------------------------------------
```

Comment:
Not a big deal, but we typically would put a blank line in here for separation.
```suggestion
}

# ------------------------------------------------------------------------------
```

Reply:
Apologies, I thought I did!  Suggestion taken and applied